python/usb_console/cmdprompt.py

```diff
@@ -49,7 +49,7 @@ def do_sc(self, args):
         args = args.split()
 
         if len(args) < 1:
-            print('Need to specify a state field to read')
+            print('Need to specify a device to command')
```

Comment:
Generally with PRs, it's better to let the author know about changes and make them handle it--that way the reviewers aren't tainted by having changes on the commit. I might have missed this if I didn't `git pull`. Thanks for doing this though!